app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -169,12 +169,12 @@ def raw_scan_job_create(entity)
   # policy_event: the event sent to automate for policy resolution
   # cb_method:    the MiqQueue callback method along with the parameters that is called
   #               when automate process is done and the event is not prevented to proceed by policy
-  def check_policy_prevent(policy_event, event_target, *cb_method)
+  def check_policy_prevent(policy_event, event_target, cb_method)
     cb = {
       :class_name  => self.class.to_s,
       :instance_id => id,
       :method_name => :check_policy_prevent_callback,
-      :args        => [*cb_method],
+      :args        => [cb_method, event_target],
```

Comment:
@moolitayer I'm assuming that `event_target` is an object (which is good)

But this is also putting the object into the queue. (which is bad)

Please change the `:args` to have the class and id.
And please change `check_policy_prevent_callback` to take arguments with a class and id.